src/components/common/NavigationButton/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ButtonHTMLAttributes } from 'react';
+import cn from '@utils/cn';
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {}
+
+export default function NavigationButton({
+  children,
+  type = 'button',
+  ...props
+}: ButtonProps) {
```

Comment:
추가적으로 작성할 타입이 없다면 `ButtonHTMLAttributes<HTMLButtonElement>`를 그대로 사용해도 좋을 것 같아요!
```suggestion
export default function NavigationButton({
  children,
  type = 'button',
  ...props
}: ButtonHTMLAttributes<HTMLButtonElement>) {
```